### PR TITLE
added module overview

### PR DESCRIPTION
**Pull request by wouterdb**
# Description

Added module landing page to docs

### CODE REVIEW

File: docs/reference/modules_content.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Modules
+
+This page gives an overview of all modules available by Inmanta on the Inmanta Orchestration Platform.
+
+
+## Adapter
+
+The following table provides and overview of all available adapters by Inmanta. Depending on the adapter they are opensource, included in specific
+product suites or directly licensed.
```
Comment (jptrindade) [inline docs/reference/modules_content.md:9]:
```suggestion
The following table provides an overview of all available adapters made by Inmanta. Depending on the adapter they can be opensource, included in specific
product suites or directly licensed.
```

File: docs/reference/modules_content.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+| athonet-ims        | Support for the full configuration of the Athonet IMS                                                         |
+| aws                | Support for AWS resources, mostly focused on EC2 and VPC                                                      |
+| aws-dc             | Support for orchestration hosted direct connect resources                                                     |
+| azure-expressroute | Support for Azure express route resources                                                                     |
+| checkpoint         | Support for Checkpoint firewalls including VSX, rules, inventory, interfaces, routing, ...                    |
+| cisco-xe           | Support for Cisco XE based devices such as CSR1000v and ASR through Netconf/YANG                              |
+| cisco-xr           | Support for Cisco XR based devices such as ASR9k, XRd and NCS through Netconf/YANG                            |
+| cloudflare         | Support for Cloudflare all resources including zero-trust services                                            |
```
Comment (jptrindade) [inline docs/reference/modules_content.md:23]:
```suggestion
| cloudflare         | Support for all Cloudflare resources including zero-trust services                                            |
```

File: docs/reference/modules_content.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+# Modules
+
+This page gives an overview of all modules available by Inmanta on the Inmanta Orchestration Platform.
```
Comment (jptrindade) [inline docs/reference/modules_content.md:3]:
```suggestion
This page gives an overview of all modules available by Inmanta on the Inmanta Service Orchestration Platform.
```

**Review by jptrindade — APPROVED** [review]
Made some suggestions about how some sentences were structured but I have no strong opinions either way.

**Review by bartv — APPROVED** [review]

**Pull request closed without merging**